README.md

```diff
@@ -1,9 +1,11 @@
 [![Build Status](https://ci.vmware.run/api/badges/vmware/docker-volume-vsphere/status.svg)](https://ci.vmware.run/vmware/docker-volume-vsphere)
 
-Docker Volume Driver for vSphere
```

Comment:
Please rebase. There should be no conflicts if you haven;t touched the file.  `git fetch; git pull --rebase master` should do the trick. Please check with @kerneltime if you need help.